foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/util/MavenUtil.java

```diff
@@ -214,6 +214,45 @@ public String getPackaging() {
             return packaging;
         }
     }
+
```

Comment:
I: This is needed so that I can retrieve profile Name from maven pom so that I can handle Old Monolith Generation and new shared data records use cases.